app/views/admin/data_exports/_form.html.slim

```diff
@@ -0,0 +1,32 @@
+= simple_form_for([:admin, @data_export.becomes(DataExport)],
+    html: { \
+      class: 'form-horizontal organization-record',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
+
+  = error_notification(f)
+  = f.input :type, as: :hidden
+  = f.input :division_id, as: :hidden
+  = f.input :name
+  = f.input :start_date
```

Comment:
I have an issue where the top of the calendar is hidden behind the header bar. @emitche have you run into this and do you have any suggestions for getting the calendar to be on top?
<img width="444" alt="Screen Shot 2019-09-24 at 11 54 18 AM" src="https://user-images.githubusercontent.com/4730344/65530318-b6e69880-dec5-11e9-8a24-4c9016aa31ef.png">